.github/workflows/pr-and-merge.yaml

```diff
@@ -0,0 +1,37 @@
+---
+name: CI
+on:
+  push:
+    branches:
+      - master
```

Comment:
The default branch name in this repo is called main and not master.

Reply:
ty!